src/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -25,6 +25,7 @@ public class HttpRequestMessage : IDisposable
         private Version _version;
         private HttpContent _content;
         private bool _disposed;
+        private bool _isConnectMethod;
```

Comment:
I don't think this needs to be a field; I think we could just make it a local in the relevant function below.